### DISCUSSION TITLE
Table Pattern Discussions/Questions

### DISCUSSION THREAD

**Discussion by mjanssen2021** [twilio-labs/paste#2102] [Office hours] [answered]
**The specific questions I for the table are...**

- What are the updates on the command bar? Are we still planning to use this pattern going forward even for saving for a table?
- What about saving a table? Is the way we are doing this for geo permissions okay?
- Is the save button meant to be persistent? or only show after an edit?
- What about editing a table? Will the alarms table edit pattern be available soon for use?
- Is the copy functionality for the sid in the table matching the patterns?
- In regards to our table - is it okay that we are using the toggle component here? Does this follow paste best practices for this component?

Background: We showed this to demo to the console team for review and they also had similar questions
Stage of design: Trying to finalize designs, some of this is being implemented now. 

Links to mockups or prototype: 
[Verify Geopermissions](https://www.figma.com/file/myHN8CnGF1BltTUh2fhVSe/2021-Q4---Verify-(Geo-Permissions)?node-id=539%3A95572)
Verify Service Settings [https://www.figma.com/file/ePGqA7BZjo8xiBS2nsUA2Y/2021-Q4-Verify-(Service-Settings-Redesign)?node-id=4%3A33884](url)

**Comment by loreina** [comment, marked as answer]
Hi @mjanssen2021 - sharing the office hours recording with you in case you'd like to revisit any of our conversations. Here's the [video](https://www.loom.com/share/217b3d7279b744b59896e3a8ed48863c) (it includes timestamps for each topic), and the password is posted in the #help-design-system channel.

Thanks for coming to office hours!
